ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/CreateScanConfigPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from 'react';
+
+import { Breadcrumb, BreadcrumbItem, Divider, PageSection, Title } from '@patternfly/react-core';
+
+import { complianceEnhancedScanConfigsBasePath } from 'routePaths';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import PageTitle from 'Components/PageTitle';
+
+import ScanConfigWizardForm from './Wizard/ScanConfigWizardForm';
+
+function ScanConfigPage(): ReactElement {
+    return (
+        <>
+            <PageTitle title="Create Scan Config" />
+            <PageSection variant="light" className="pf-u-py-md">
+                <Breadcrumb>
+                    <BreadcrumbItemLink to={complianceEnhancedScanConfigsBasePath}>
+                        Scan schedules
+                    </BreadcrumbItemLink>
+                    <BreadcrumbItem isActive>Create scan schedule</BreadcrumbItem>
+                </Breadcrumb>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                <Title headingLevel="h1" className="pf-u-py-lg pf-u-px-lg">
+                    Create Scan Schedule
```

Comment:
What do you think about sentence case **Create scan schedule** similar to elsewhere
![Create_collection](https://github.com/stackrox/stackrox/assets/11862657/f57251f1-545d-4c78-906c-b00576e4cfad)

https://www.patternfly.org/ux-writing/capitalization#capitalization-across-patternfly

Also can `PageTitle` and `Title` elements have consistent text?

Thinking ahead, will is this wizard only for **create** action or also for **edit** (and **clone** if needed).